dotCMS/src/curl-test/VersionableResource.postman_collection.json

```diff
@@ -0,0 +1,602 @@
+											"value": "admin@dotcms.com",
+											"type": "string"
+										},
+										{
+											"key": "saveHelperData",
+											"type": "any"
+										},
+										{
+											"key": "showPassword",
+											"value": false,
+											"type": "boolean"
+										}
+									]
+								},
+								"method": "POST",
+								"header": [],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \"title\":\"PostMan Test\",\n    \"friendlyName\":\"This is a postman test description\",\n    \"body\":\"This is the body\"\n}",
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "{{serverURL}}/api/v1/templates",
+									"host": [
+										"{{serverURL}}"
+									],
+									"path": [
+										"api",
+										"v1",
+										"templates"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Publish Template Success",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"var jsonData = pm.response.json();",
+											"pm.test(\"Status code should be ok 200\", function () {",
+											"    pm.response.to.have.status(200);",
+											"});",
+											"",
+											"pm.test(\"Publish Template Check\", function () {",
+											"    pm.expect(jsonData.entity.successCount).to.be.eql(1);",
+											"});",
+											"",
+											"pm.test(\"Failed To Publish Template Check\", function () {",
+											"    pm.expect(jsonData.entity.fails).to.have.lengthOf(0);",
+											"});",
+											""
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"auth": {
+									"type": "basic",
+									"basic": [
+										{
+											"key": "password",
+											"value": "admin",
+											"type": "string"
+										},
+										{
+											"key": "username",
+											"value": "admin@dotcms.com",
+											"type": "string"
+										},
+										{
+											"key": "saveHelperData",
+											"type": "any"
+										},
+										{
+											"key": "showPassword",
+											"value": false,
+											"type": "boolean"
+										}
+									]
+								},
+								"method": "PUT",
+								"header": [],
+								"body": {
+									"mode": "raw",
+									"raw": "[\"{{templateIdToDeleteVersion}}\"]\n",
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "{{serverURL}}/api/v1/templates/_publish",
+									"host": [
+										"{{serverURL}}"
+									],
+									"path": [
+										"api",
+										"v1",
+										"templates",
+										"_publish"
+									]
+								},
+								"description": "Publish a template successfully"
+							},
+							"response": []
+						},
+						{
+							"name": "Edit Template Success",
```

Comment:
There are 2 tests with name "Edit Template Success". Is the test duplicated?

Reply:
Yes it's duplicated but it's intended. Is to have 1 version published, one version working and one version to delete successfully.